### PR TITLE
Fix: [autobuy] fix error when bollinger settings is not set

### PR DESCRIPTION
**Pull request by narumiruna**

### CODE REVIEW

**Review by c9s — APPROVED** [review]

File: pkg/strategy/autobuy/strategy.go [hunk 1]
```diff
@@ -64,27 +65,57 @@ func (s *Strategy) InstanceID() string {
...
+
+		if s.Bollinger.BandWidth <= 0 {
+			return fmt.Errorf("bollinger band width must be greater than 0")
+		}
+	}
 	return nil
 }
 
 func (s *Strategy) Defaults() error {
+	if s.OrderType == "" {
```
Comment (c9s) [inline pkg/strategy/autobuy/strategy.go:97]: order type needs to be aligned with the price type

if price type = taker, the default limit maker won’t work

Comment (narumiruna) [reply pkg/strategy/autobuy/strategy.go:97]: so add a warning here?

Comment (c9s) [reply pkg/strategy/autobuy/strategy.go:97]: the default order type should be "limit", because it would work for all the cases

Comment (narumiruna) [reply pkg/strategy/autobuy/strategy.go:97]: I will set the default order type to limit

**Pull request merged**